Client/components.json

```diff
@@ -67,5 +67,10 @@
         "no smiles": ["float", "smiles"],
         "comparator": ["choice_propagate", ">", ">=", "<", "<=", "=="],
         "threshold": ["float", "equal"]
+    },
+    "People": {
+        "hasPeople": ["choice_propagate", "true", "false"],
+        "minPeople": ["int", "true"],
+        "maxPeople": ["int", "true"]
```

Comment:
You need to either change the input's label of "int" or rewrite if handler to accept both your type with parameter "true" and another type with min-max range values